plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -405,16 +405,14 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
      * @throws Exception
      */
     @RequirePOST
-    public void doExportJSONSchema(StaplerRequest req, StaplerResponse res) throws Exception {
+    public void doExportNewJSONSchema(StaplerRequest req, StaplerResponse res) throws Exception {
 
         if (!Jenkins.getInstance().hasPermission(Jenkins.ADMINISTER)) {
             res.sendError(HttpServletResponse.SC_FORBIDDEN);
             return;
         }
-        writeJSONSchema();
-        res.setContentType("application/x-yaml; charset=utf-8");
-        res.addHeader("Content-Disposition", "attachment; filename=JSONSchema.json");
-        export(res.getOutputStream());
+        res.setContentType("application/json; charset=utf-8");
+        res.getWriter().write(writeJSONSchema());
```

Comment:
@casz So you are saying that this would not create a URL ExportNewJSONSchema, and isnt write supposed be writing, I tried creating a `ViewNewSchema` but that doesnt seem to write as well